astropy/stats/sigma_clipping.py

```diff
@@ -634,24 +655,29 @@ def sigma_clip(data, sigma=3, sigma_lower=None, sigma_upper=None, maxiters=5,
 
     Clipped (rejected) pixels are those where::
 
-        data < cenfunc(data [,axis=int]) - (sigma_lower * stdfunc(data [,axis=int]))
-        data > cenfunc(data [,axis=int]) + (sigma_upper * stdfunc(data [,axis=int]))
+        data < center - (sigma_lower * std)
+        data > center + (sigma_upper * std)
+
+    where::
+
+        center = cenfunc(data [, axis=])
+        std = stdfunc(data [, axis=])
 
     Invalid data values (i.e., NaN or inf) are automatically clipped.
 
     For an object-oriented interface to sigma clipping, see
     :class:`SigmaClip`.
 
     .. note::
-        `scipy.stats.sigmaclip`
-        provides a subset of the functionality in this class.  Also, its
-        input data cannot be a masked array and it does not handle data
-        that contains invalid values (i.e., NaN or inf).  Also note that
-        it uses the mean as the centering function.
-
-        If your data is a `~numpy.ndarray` with no invalid values and
-        you want to use the mean as the centering function with
-        ``axis=None`` and iterate to convergence, then
+        `scipy.stats.sigmaclip` provides a subset of the functionality
+        in this class. Also, its input data cannot be a masked array
+        and it does not handle data that contains invalid values (i.e.,
+        NaN or inf). Also note that it uses the mean as the centering
+        function.
+
+        If your data is a `~numpy.ndarray` with no invalid values
+        and you want to use the mean as the centering function
+        with ``axis=None`` and iterate to convergence, then
         `scipy.stats.sigmaclip` is ~25-30% faster than the equivalent
```

Comment:
Is this still true with the new gufunc?

Reply:
I did some new profiling, and it's a bit hard to summarize in a single sentence.  Sometimes astropy is *faster* than scipy:

<pre>
size     % slower
1e8.     -6.9
1e7.     11.5
1e6.      6.3
1e5.     36.1
1e4.     10.6
1e3.    -18.2
</pre>

Perhaps we should just remove the part about the performance, and instead simply include the equivalent keywords to match `scipy.stats.sigmaclip`?

Reply:
Yes, sounds good! Though fine to do in a separate PR, if you prefer.

Reply:
I went ahead and did it here.  Thanks.